chapters/building-blocks/overview/reading/README.md

```diff
@@ -1,17 +1,47 @@
 ## Building Blocks
 
-Owner: Alex Apostolescu
-Reviewer: Ștefan Jumărea
+The top-level approach for building materials for a subject involves four key components:
 
-Present the high-level organization of a course or a subject that will be part of a repository:
+- **The content repository**
 
-1. A public content repository, organized and used as shown in [the "Developing and Organizing Content" section](../../../develop-organize/overview/reading/README.md).
-1. A private repository, with reference solutions and other items we want to keep private.
-1. An implementation-specific repository, used by the organization or person who does the actual content delivery, organized as detailed in [the "Using and Delivering Content" section](../../../use-deliver/overview/reading/README.md).
-1. Support infrastructure for storing, publishing, validating content and providing support in education activities (quiz generation, automatic submission checking etc.), detailed in [the "Support Infrastructure" session](../../../infrastructure/overview/reading/README.md).
+  This is a public repository that serves as a storing point for educational materials.
+  It follows the structure outlined in the ["Developing and Organizing Content" section](../../../develop-organize/overview/reading/README.md) and does not place restrictions on how information is presented.
 
-Of relevance are the roles of those who manage, develop or contribute to content, detailed in [the "Roles" section](../../../roles/overview/reading/README.md).
+- **The delivery repository**
 
-And the guidelines for maintaining content, presented in [the "Curating and Maintaining Content" section](../../../curate-maintain/overview/reading/README.md).
+  This repository is an extension of **the content repository** and is dedicated to structuring and customizing content for delivery.
+  It follows the guidelines in the ["Developing and Organizing Content" section](../../../develop-organize/overview/reading/README.md).
 
-Conventions used in this methodology are detailed in [the "Conventions" section](../../conventions/reading/README.md).
+- **The solutions repository**
+  
```

Comment:
```suggestion

```

Reply:
This is intended. WIthout this, the description comes in-line instead of being a nested subparagraph.